ui/partials/logviewer/lvLogLines.html

```diff
@@ -2,9 +2,10 @@
      ng-if="loading"> Loading... </div>
 
 <div ng-repeat="lv_line in displayedLogLines"
-     ng-class="{'text-danger': (lv_line.hasError == true),
-                'lv-selected-lines': (displayedStep.order === lv_line.index)}"
      class="lv-log-line"
      line="{{ ::lv_line.index }}">
-  <span class="lv-line-text">{{ ::lv_line.text }}</span>
+     <div class="lv-line-num" ng-click="click(lv_line, $event)"> {{::lv_line.index}}  </div>
+     <div ng-class="{'text-danger': (lv_line.hasError == true),
+                     'lv-selected-lines': (lv_line.index >= selectedBegin && lv_line.index <= selectedEnd)}"
+            class="lv-line-text"> {{  ::lv_line.text}}</div>
```

Comment:
This is just inherited code, but maybe let's try removing the leading whitespace inside the template braces here and on line 6. I tried it locally and it doesn't seem to make any difference in the rendered output.

`{{::lv_line.index}}`
`{{::lv_line.text}}`

I defer to @camd though, if he has a preference. Angular's docs tend to imply no spacing.